src/views/PurchaseOrder.vue

```diff
@@ -96,6 +98,12 @@ export default defineComponent({
     methods: {
       getFile(event) {
         this.file = event.target.files[0];
+        if(this.file){
+          showToast(translate("File uploaded successfully"));
```

Comment:
We should move `this.parseFile()` inside if, as we only want to parse the file when we have it, otherwise not.

Reply:
Updated the code.